### ISSUE TITLE
DropDown: In Chrome+ NVDA, dropdown option selection causes NVDA to read title of page

### ISSUE THREAD

**Issue by shsengar** [OfficeDev/office-ui-fabric-react#11130]
### Describe the issue:
Drop down component of office fabric starts reading title of home page after selecting option from dropdown.

### Please provide a reproduction of the issue in a codepen:
1)Open Google chrome
2) Go to https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown
3) Start NVDA as screen reader on Customized DropDown section.

![record8](https://user-images.githubusercontent.com/57530569/68478945-1ec72780-0257-11ea-9913-92cf67f831c4.gif)

#### Actual behavior:
It is reading the title of page along with the option selected

#### Expected behavior:
It should only tell read out the relevant information. It may confuse the screenreader users.

**Comment by JasonGore** [comment]
This seems to only happen under very specific conditions. It only appears to happen when using 'enter' to select an item, and does not happen when using 'space' or the clicking with the mouse. It also only occurs with Chrome + NVDA. It does not happen with MS Edge + NVDA and Firefox + NVDA.

Given the specific nature of this issue, I think this should be reported against Chrome or NVDA.
